storage/internal/elastic/es_utils.go

```diff
@@ -178,7 +178,7 @@ func doQueryEs(c *elasticsearch6.Client, conf elasticStoreConf,
 		c.Search.WithSort("iid:"+order),
 	)
 	if e != nil {
-		err = errors.Wrapf(err, "Failed to perform ES search on index %s, query was: <%s>, error was: %+v", index, query, err)
+		err = errors.Wrapf(e, "Failed to perform ES search on index %s, query was: <%s>, error was: %+v", index, query, err)
```

Comment:
Should use the error `e` instead of `err` as well at the end of line (last argument)